src/main/java/com/madadata/eval/prestoconnectivity/resource/MyResource.java

```diff
@@ -66,4 +104,26 @@ public Object tableMeta() throws Exception {
                 "table_catalog", "table_schema", "table_name", "table_type");
     }
 
+    @GET
+    @Path("/create")
+    @Produces(MediaType.APPLICATION_JSON)
+    public int create(@QueryParam("name") @NotEmpty String name) throws Exception {
+        return jdbiCreate(String.format("create table %s (id int primary key, name varchar(100))", name));
+    }
+
+    @GET
+    @Path("/create-table")
+    @Produces(MediaType.APPLICATION_JSON)
+    public int createTable(@QueryParam("name") @NotEmpty String name) throws Exception {
+        return executeCreate(String.format("create table %s (id integer, name varchar(100))", name));
```

Comment:
just so that you know, don't ever do this in real projects - it's suspect to SQL-injection